pydicom/tests/testing.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+"""Utility functions to help with testing."""
+
+from re import compile
+
+import pytest
+
+
+def assert_raises_regex(type_error, message, func, *args, **kwargs):
```

Comment:
No need with pytest. see [this](https://github.com/pydicom/pydicom/pull/488/#issuecomment-323736235)